esx_service/cli/vmdkops_admin.py

```diff
@@ -255,6 +255,20 @@ def commands():
         'status': {
             'func': status,
             'help': 'Show the status of the vmdk_ops service'
+        },
+        'set': {
+            'func': set_vol_opts,
+            'help': 'Edit settings for a given volume',
+            'args': {
```

Comment:
Sure. Add a note in comment that change will be effective next time volume is mounted or something on those lines.